### ISSUE TITLE
[Bug]: package saves timezones with spaces instead of underscores

### ISSUE THREAD

**Issue by ceejayoz** [omar-haris/filament-timezone-field#3]
### What happened?

When selecting the "America / New York" timezone, the value saved to the database is `America/New York`.

The correct timezone for this (and other similar timezones with a space, like `Asia/Ho_Chi_Minh`) is `America/New_York`.

### How to reproduce the bug

Select a timezone with a space. Save to database. Attempt to use it with something like `now($timezone)` and receive an error of ` Unknown or bad timezone (America/New York) `.

### Package Version

1.0.3

### PHP Version

PHP 8.3.12

### Laravel Version

11.25.0

**Comment by ceejayoz** [comment]
As a temporary workaround:

```php
Timezone::make('timezone')
  ->formatStateUsing(fn($state) => Str::of($state)->replace('_', ' '))
  ->dehydrateStateUsing(fn($state) => Str::of($state)->replace(' ', '_'))
```

**Comment by HayderHatem** [comment]
> As a temporary workaround:
> 
> ```
> Timezone::make('timezone')
>   ->formatStateUsing(fn($state) => Str::of($state)->replace('_', ' '))
>   ->dehydrateStateUsing(fn($state) => Str::of($state)->replace(' ', '_'))
> ```

instead of wasting your time looking for a workaround, just edit translation file 
[#2](https://github.com/omar-haris/filament-timezone-field/issues/2#issuecomment-2028829267)

**Comment by omar-haris** [comment]
solved [#4 ](https://github.com/omar-haris/filament-timezone-field/pull/4).

**Comment by ceejayoz** [comment]
Thanks @omar-haris!
